docs/database/database-metrics.md

```diff
@@ -35,6 +37,132 @@ and attributes but more may be added in the future.
 > until a transition plan to the (future) stable semantic conventions has been published.
 > Conventions include, but are not limited to, attributes, metric and span names, and unit of measure.
 
+## Database operation
+
+### Metric: `db.client.operation.duration`
+
+**Status**: [Experimental][DocumentStatus]
+
+This metric is [required][MetricRequired].
+
+When this metric is reported alongside a database operation span, the metric value SHOULD be the same as the database operation span duration.
+
+This metric SHOULD be specified with
+[`ExplicitBucketBoundaries`](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.26.0/specification/metrics/api.md#instrument-advisory-parameters)
+of `[ 0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10 ]`.
```

Comment:
Is this necessary? It appears to be a default configuration, so I wouldn't expect to see it here.

Reply:
the default configuration is in milliseconds 😭

Reply:
I think we could do better job in the tooling defining, referencing, and generating boundaries in md and code - created https://github.com/open-telemetry/build-tools/issues/291 to track it